### PR TITLE
Fibonacci

### PR DESCRIPTION
**Pull request by michantoniuk**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: fibonacci
🏅 Granted 6 XP for @michantoniuk

**Comment by ziobron** [comment]
⏱ 3 XP extra for @michantoniuk for on-time delivery

File: homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,21 @@
 #pragma once
 
 int fibonacci_iterative(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:5]: Sequence hold exactly the same value as you want to return for 0 and 1 :)

Better write:

`if (sequence < 2) {
    return sequence;
}`

You avoid unnecessary calculations.

File: homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+
+    return fib_top;
 }
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if (sequence <= 1)
+        return sequence;
+    return fibonacci_recursive(sequence-1) + fibonacci_recursive(sequence-2);
```
Comment (nauka-programowania-MA) [inline homework/fibonacci/fibonacci.hpp:20]: Add space between operations like '+' '-' '!=' '==' etc...

**Review by nauka-programowania-MA — APPROVED** [review]
Perfect! I really enjoy your solutions!

**Pull request closed without merging**